engine/runtime-compiler/src/main/scala/org/enso/compiler/data/BindingsMap.scala

```diff
@@ -23,7 +23,7 @@ import scala.annotation.unused
   */
 
 @SerialVersionUID(
-  8160L // Use BindingsMap
+  9057L // Use BindingsMap
```

Comment:
We are no longer using `serialVersionUID` as we have our own `Persistance` framework. Instructions to [version caches are here](https://github.com/enso-org/enso/blob/f2d2f73e8941b5cfaaa3edfdfe0c05f4b8117d47/docs/runtime/ir-caching.md#portability-and-versioning).

Sorry for not stating that more clearly when requesting you to _"Change caches version ID"_, but the last change to the documentation was [done by you](https://github.com/enso-org/enso/blame/f2d2f73e8941b5cfaaa3edfdfe0c05f4b8117d47/docs/runtime/ir-caching.md#L170), so I expected you to remember the context.

As a result of not versioning the caches properly, I have to clean `rm -rf ~/.local/shared/enso/caches/ir` to fix 
- #8941